app/models/modernised/PropertyLinkStatus.scala

```diff
@@ -27,5 +29,5 @@ object PropertyLinkStatus extends Enumeration {
   val REVOKED = Value("REVOKED")
   val MORE_EVIDENCE_REQUIRED = Value("MORE_EVIDENCE_REQUIRED")
 
-  implicit val format = enumFormat(PropertyLinkStatus)
+  implicit val format: Format[modernised.PropertyLinkStatus.Value] = enumFormat(PropertyLinkStatus)
```

Comment:
Not a functional issue but could this just be Format[PropertyLinkStatus.Value]?

Reply:
Yeah it definitely can, I must've missed that 👍